ReactiveUsersMicroservice/src/main/java/Cloud2024/ReactiveUsersMicroservice/Infrastructure/UserMapper.java

```diff
@@ -22,6 +22,7 @@ public ModelMapper modelMapper(){
                     mapper.map(src -> generateRandomAsterisks(), UserBoundary::setPassword);
                     mapper.map(UserEntity::getFirstName, (dest, v) -> dest.getName().setFirst((String) v));
                     mapper.map(UserEntity::getLastName, (dest, v) -> dest.getName().setLast((String) v));
+                    mapper.map(UserEntity::getDepartment, UserBoundary::setDepartment); // Map department
```

Comment:
The department property should be map autos like all the other properties.
Here you see only properties that need customization to their mapping.